frontend/components/groups/js/groups.service.ts

```diff
@@ -67,6 +66,23 @@ export class GroupsService {
 			});
 	}
 
+	// Helper  for searching strings
+	public stringSearch(superString, subString) {
+		if (!superString) {
+			return false;
+		}
+
+		return (superString.toLowerCase().indexOf(subString.toLowerCase()) !== -1);
+	}
+
+	public groupsFilterSearch(searchQuery: string): any[] {
+
+		const groupSearchResults = this.state.groups.filter((group) => {
```

Comment:
avoid assigning a variable then immediately return it.

just return the whole thing! (this is probably James' fault... lol)